src/main/kotlin/io/snyk/plugin/services/SnykTaskQueueService.kt

```diff
@@ -156,7 +156,6 @@ class SnykTaskQueueService(val project: Project) {
         ) {
             override fun run(indicator: ProgressIndicator) {
                 val toolWindowPanel = project.service<SnykToolWindowPanel>()
-                if (toolWindowPanel.currentIacResult != null) return
```

Comment:
That will lead to at least IaC scan run every time when scan is requested, even if no IaC related files was changed. Check with Georgy if that's acceptable.

Reply:
Yeah, I think it is what's expected, but I'll confirm with Georgi & Andy.

Reply:
afaik avoiding such unneeded scans was the reason why we invented caches...